lib/saml_idp/xml_security.rb

```diff
@@ -237,6 +236,13 @@ def verify_signature(base64_cert, sig_alg, signature, canon_string, soft)
         cert = OpenSSL::X509::Certificate.new(cert_text)
         signature_algorithm = algorithm(sig_alg)
 
+        if signature_algorithm != OpenSSL::Digest::SHA256
```

Comment:
Should this be checking if the signature algorithm is equal to the [configured one](https://github.com/18F/saml_idp/pull/124/files#diff-67678800438a2f1aaaeb42c42eaed5f503fd9307b3b7873ddf1f7115bb74e35bR13) rather than a static one?

Reply:
i think that makes sense, i'll update

Reply:
updated in https://github.com/18F/saml_idp/pull/124/commits/0e15f5d0dd628ddf74b586aa67288465ea933fb3